docs/oauth2-oidc/jwt-access-token.mdx

```diff
@@ -46,13 +46,30 @@ access token has expired. Refresh tokens are always opaque tokens in Ory because
 
 ## JWT access tokens
 
-To configure Ory OAuth2 to issue JWT access tokens, you'll need to update the configuration as Ory defaults to opaque tokens:
+### Global configuration
+
+To configure Ory OAuth2 to issue JWT access tokens for all clients, you'll need to update the configuration as Ory defaults to
+opaque tokens:
 
 ```shell
 ory patch oauth2-config {project.id} \
   --replace "/strategies/access_token=\"jwt\""
 ```
 
+### Per-client configuration
+
+To configure Ory OAuth2 to issue JWT access tokens for a specific client, you'll need to update the client configuration as Ory
+defaults to opaque tokens:
+
+```mdx-code-block
+import CodeBlock from '@theme/CodeBlock'
+import setAccessTokenStrategyExample from '!!raw-loader!../../code-examples/sdk/typescript/src/oauth2/client-set-access-token-strategy.ts'
+
+<CodeBlock language="tsx">{setAccessTokenStrategyExample}</CodeBlock>
```

Comment:
Couldn't this also be done through cURL and Go? Maybe we should add it as well?
https://github.com/ory/docs/tree/master/code-examples/curl
https://github.com/ory/docs/tree/master/code-examples/sdk/go

Reply:
Let's do this in a follow-up PR to have something shipped right now